geopy/geocoders/geonames.py

```diff
@@ -125,6 +139,7 @@ def reverse(
             query,
             exactly_one=DEFAULT_SENTINEL,
             timeout=DEFAULT_SENTINEL,
+            find_nearby=False,
```

Comment:
The definitely shouldn't be a boolean, because there're other reverse methods available in the service's API, such as `findNearbyPostalCodes`. It should be a choice-like string.